src/components/files/Listing.vue

```diff
@@ -534,9 +534,12 @@ export default {
       return checkMimeType(file.type) || checkDocumentPreview(file.type);
     },
     preview: function (file) {
-      this.$store.commit('pushFolder', { id: file.id, name: file.name });
-      this.$store.commit('setReload', true);
-
+        if (api.isDocumentEdit(file.type)) {
+        window.open(api.previewDocs(file.id));
```

Comment:
I think it will be better if the function preview docs will open the docs itself. Otherwise, the function name is not good (the function only returns the url and not preview).